src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const appendRowBtn = document.querySelector('.append-row');
+const removeRowBtn = document.querySelector('.remove-row');
+const appendColumnBtn = document.querySelector('.append-column');
+const removeColumnBtn = document.querySelector('.remove-column');
+let rowCount = table.rows.length;
+let columnCount = table.rows[0].cells.length;
+
+if (rowCount === 10) {
+  appendRowBtn.disabled = true;
+}
+
+if (rowCount === 2) {
+  removeRowBtn.disabled = true;
+}
+
+if (columnCount === 10) {
+  appendColumnBtn.disabled = true;
+}
+
+if (columnCount === 2) {
+  removeColumnBtn.disabled = true;
+}
+
+appendRowBtn.addEventListener('click', function() {
+  if (rowCount < 10) {
+    const row = table.insertRow();
+
+    for (let i = 0; i < columnCount; i++) {
+      row.insertCell();
+    }
+    rowCount++;
+    removeRowBtn.disabled = false;
+
+    if (rowCount === 10) {
+      appendRowBtn.disabled = true;
+    }
```

Comment:
Seems that it's useless. Look, you check if `rowCount < 10` before, so in this case the rowCount can't be equal to 10. The same logic is in the other functions. Check if it'll work when you just delete it
```suggestion
```